From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Bartosz Bilas <b.bilas@grinn-global.com>
Date: Mon, 4 Aug 2025 15:50:50 +0200
Subject: arm: dts: grinn: move auxadc node into board device tree

For the upcoming support of boards based on the Genio 510,
move the auxadc node out of the generic grinn-genio-sbc.dtsi file,
as this peripheral is not available on Genio 510.
---
 arch/arm/dts/grinn-genio-700-sbc.dts | 4 ++++
 arch/arm/dts/grinn-genio-sbc.dtsi    | 4 ----
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/dts/grinn-genio-700-sbc.dts b/arch/arm/dts/grinn-genio-700-sbc.dts
index 111111111111..222222222222 100644
--- a/arch/arm/dts/grinn-genio-700-sbc.dts
+++ b/arch/arm/dts/grinn-genio-700-sbc.dts
@@ -22,3 +22,7 @@
 		reg = <0 0x40000000 1 0x00000000>;
 	};
 };
+
+&auxadc {
+	status = "okay";
+};
diff --git a/arch/arm/dts/grinn-genio-sbc.dtsi b/arch/arm/dts/grinn-genio-sbc.dtsi
index 111111111111..222222222222 100644
--- a/arch/arm/dts/grinn-genio-sbc.dtsi
+++ b/arch/arm/dts/grinn-genio-sbc.dtsi
@@ -203,10 +203,6 @@
 	status = "okay";
 };
 
-&auxadc {
-	status = "okay";
-};
-
 &u3phy0 {
 	status = "okay";
 };
-- 
Armbian

